google-cloud-util/google-cloud-compat-checker/README.md

```diff
@@ -23,7 +23,19 @@ OS details:
   os.detected.classifier: linux-x86_64
   os.detected.release: ubuntu
   os.detected.release.version: 14.04
+JVM details:
+  Java version: 1.8.0_144-b01
+  Java specification version: 1.8
```

Comment:
We would build logic on the specification version, but probably not the full java version, but I think it's useful to have some extra details in the output just for debugging purposes.